### PR TITLE
Fix pyraise

### PR DESCRIPTION
**Pull request by tkf**
fix https://github.com/JuliaPy/pyjulia/pull/262#issuecomment-479778182

### CODE REVIEW

File: src/exception.jl [hunk 1]
```diff
@@ -180,9 +180,9 @@ function pyraise(e::PyError, ::Vector = [])
     ccall((@pysym :PyErr_Restore), Cvoid, (PyPtr, PyPtr, PyPtr),
           e.T, e.val, e.traceback)
     # refs were stolen
-    setfield!(e.T, :o, C_NULL)
-    setfield!(e.val, :o, C_NULL)
-    setfield!(e.traceback, :o, C_NULL)
+    setfield!(e.T, :o, PyPtr_NULL)
+    setfield!(e.val, :o, PyPtr_NULL)
+    setfield!(e.traceback, :o, PyPtr_NULL)
```
Comment (tkf) [inline src/exception.jl]: They are all `PyObject` so `.o` has to be of type `PyPtr`.

Comment (stevengj) [reply src/exception.jl]: This is fine, but I thought `setfield!` automatically called `convert` (JuliaLang/julia#16195)?

**Comment by stevengj** [comment]
The AppVeyor failure
```
Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0xffdefdd -- unknown function (ip: 0FFDEFDD)
in expression starting at C:\projects\pycall-jl-nu3aa\test\runtests.jl:29
unknown function (ip: 0FFDEFDC)
PyObject_GetAttr at C:\windows\SYSTEM32\python27.dll (unknown line)
PyObject_CallFunctionObjArgs at C:\windows\SYSTEM32\python27.dll (unknown line)
PyType_Modified at C:\windows\SYSTEM32\python27.dll (unknown line)
PyCFunction_Call at C:\windows\SYSTEM32\python27.dll (unknown line)
PyObject_Call at C:\windows\SYSTEM32\python27.dll (unknown line)
PyObject_CallFunctionObjArgs at C:\windows\SYSTEM32\python27.dll (unknown line)
PyObject_IsInstance at C:\windows\SYSTEM32\python27.dll (unknown line)
pyint_query at C:\projects\pycall-jl-nu3aa\src\PyCall.jl:173
pytype_query at C:\projects\pycall-jl-nu3aa\src\conversions.jl:770
pytype_query at C:\projects\pycall-jl-nu3aa\src\conversions.jl:803
convert at C:\projects\pycall-jl-nu3aa\src\conversions.jl:828
roundtrip at C:\projects\pycall-jl-nu3aa\test\runtests.jl:13 [inlined]
roundtrip at C:\projects\pycall-jl-nu3aa\test\runtests.jl:14 [inlined]
roundtripeq at C:\projects\pycall-jl-nu3aa\test\runtests.jl:16
```
is surprising…

File: src/exception.jl (hunk 1 above)
Comment (tkf) [reply src/exception.jl]: It looks like that's not the case https://github.com/JuliaLang/julia/issues/16195#issuecomment-480089197

**Pull request merged**